sdk/quantum/quantum-jobs/test/public/quantumJobClient.spec.ts

```diff
@@ -1,16 +1,160 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import * as assert from "assert";
+import { ContainerClient, BlockBlobClient } from "@azure/storage-blob";
 import { QuantumJobClient } from "../../src";
+import { authenticate } from "../utils/testAuthentication";
+import { replaceStorageSig } from "../utils/recorderUtils";
+import { Recorder } from "@azure/test-utils-recorder";
+import chai from "chai";
+import * as fs from "fs";
+import { TokenCredential } from "@azure/identity";
+import { isPlaybackMode } from "@azure/test-utils-recorder";
 
-describe("Quantum Job Client tests", function() {
-  it("can create the client object successfully", () => {
-    const obj: QuantumJobClient = new QuantumJobClient(
-      "sample-subscription-id",
-      "sample-resource-group-name",
-      "sample-workspace-name"
-    );
-    assert.notEqual(obj, null);
+const assert = chai.assert;
+
+describe("Quantum job lifecycle", () => {
+  let client: QuantumJobClient;
+  let recorder: Recorder;
+  let credentials: TokenCredential;
+
+  beforeEach(async function() {
+    const authentication = await authenticate(this);
+    client = authentication.client;
+    recorder = authentication.recorder;
+    credentials = authentication.credentials;
+  });
+
+  afterEach(async function() {
+    await recorder.stop();
+  });
+
+  it("Test Get Providers Status", async function() {
+    let index = 0;
+    for await (const status of client.providers.listStatus()) {
+      assert.isNotEmpty(status.id);
+      assert.isNotNull(status.targets);
+      assert.isNotNull(status.currentAvailability);
+      ++index;
+    }
+    // Should have at least one in the list.
+    assert.isTrue(index >= 1);
```

Comment:
Can we check for a specific index value rather than >= 1?

Reply:
That can vary per workspace and per service side changes, so, for now that is the best we can test on the client side